perf-dash.k8s.io/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
```

Comment:
I think we can start from this. If needed, Matt or myself will be able to add others.